src/EPiServer.SocialAlloy.Web/Social/Repositories/Moderation/CommunityMembershipModerationRepository.cs

```diff
@@ -355,27 +356,27 @@ public void Moderate(string workflowId, string action, string userId, string com
         /// <returns>Moderation workflow supporting the specified group</returns>
         private CommunityMembershipWorkflow GetWorkflowFor(string group)
         {
-            CommunityMembershipWorkflow expectedSocialWorkflow = null;
-            IEnumerable<Composite<Workflow, MembershipModeration>> listOfWorkflow = Enumerable.Empty<Composite<Workflow, MembershipModeration>>();
-
-            var filterWorkflowsByGroup =
-                FilterExpressionBuilder<MembershipModeration>.Field(m => m.Group)
-                                                             .EqualTo(group);
-
-            var criteria = new CompositeCriteria<WorkflowFilter, MembershipModeration>
-            {
-                PageInfo = new PageInfo { PageSize = 1 },
-                ExtensionFilter = filterWorkflowsByGroup
-            };
-
             try
             {
-                listOfWorkflow = this.workflowService.Get(criteria).Results;
-                if (listOfWorkflow.Count() > 0)
+                IEnumerable<Workflow<MembershipModeration>> workflows = Enumerable.Empty<Workflow<MembershipModeration>>();
+
+                var workflowFilters = new FieldExpressionBuilder<MembershipModeration>();
```

Comment:
ah crap...forgot to come back to this one...yeah, you're right...much simpler this way :-)